corehq/apps/linked_domain/static/linked_domain/js/domain_links.js

```diff
@@ -66,33 +67,50 @@ hqDefine("linked_domain/js/domain_links", [
             }
         }
 
+        self.domain_links = ko.observableArray(_.map(data.linked_domains, function (link) {
+            return DomainLink(link);
+        }));
+
+        // pull content
+        self.model_status = _.map(data.model_status, ModelStatus);
         self.can_update = data.can_update;
         self.models = data.models;
 
-        self.model_status = _.map(data.model_status, ModelStatus);
+        // manage downstream domains tab
+        self.paginated_domain_links = ko.observableArray([]);
```

Comment:
This file is inconsistent, but the js naming convention is camel case for attributes.

Reply:
Realized I never replied. I've gone back and fourth on updating existing variables but it results in enough changes to warrant its own PR. This is a good reminder to follow the js naming convention for any variables that I add though so thanks.